main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import
+	"fmt"
+	"os"
+
+"github.com/aws/aws-sdk-go/aws"
+"github.com/aws/aws-sdk-go/aws/session"
+"github.com/aws/aws-sdk-go/service/dynamodb"
+"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+
+
+// Define a struct to hold the item data
+type Item struct {
+	ID   int    `json:id`
+	Name string `json:name`
+}
+
+func main() {
+	// Create a session with AWS
+	sess, err = session.NewSession(&aws.Config{
+		Region: aws.String("australia"),
+	})
+
+	if err != nil {
+		fmt.Println("Error creating session:", err)
+	}
+
+	// Create a DynamoDB client
+	svc = dynamodb.New(sess)
+
+	// Create an item to write
+	item = Item{
+		ID:   1,
+		Name: "Bing",
+	}
+
+	// Convert the item to a map of attribute values
+	av, err = dynamodbattribute.MarshalMap(item)
```

Comment:
The section of the code is mostly plumbing and breaks the flow of comprehending the intent of the code, could it be extrtact to a createItem func  which would hide it?